dnf/yum/rpmtrans.py

```diff
@@ -138,10 +138,8 @@ def filelog(self, package, action):
         self.rpm_logger.info(msg)
 
 class RPMTransaction(object):
-    def __init__(self, base, test=False, display=TransactionDisplay):
-        self.display = display
-        if isinstance(display, collections.Callable):
-            self.display = display()
+    def __init__(self, base, test=False, displays=()):
```

Comment:
I also probably need to restore the creation of the `TransactionDisplay` instance if the sequence is empty.